### ISSUE TITLE
default ignore settings break select validation

### ISSUE THREAD

**Issue by njbelf** [jzaefferer/jquery-validation#1655]
Jquery 2.1.4
Jquery.validation.js 1.14

I was pulling my hair out (what little I have left) over this one.   For some reason the default ignore setting of ":hidden" causes selects to not be validated.   Removing this default setting fixes the issue.

**Comment by Arkni** [comment]
Can you provide a jsfiddle describing the issue ?

**Comment by njbelf** [comment]
Not at this time.. too busy with work..

**Comment by staabm** [comment]
@njbelf will close this issue until we get a fiddle to re-produce. without a example we cannot do anything for you.

**Comment by njbelf** [comment]
JsFiddle does not support this combination of JQuery 2.1.4 and
jqueryvalidation.js of 1.14.

    This combination breaks ALL of our selects on our website so it should
be fairly easy to reproduce.

However, when I try to set up a JSFiddle, it breaks jsfiddle :
https://jsfiddle.net/p80q2rLs/

Here is the html:

<form id='QuickCalc' name='QuickCalc' method='post' action=''>
    <select id='FirstSelect' name='FirstSelect'>
        <option value>--Select--</option>
        <option value="1">1</option>
        <option value="2">2</option>
        <option value="3">3</option>
    </select>
    <input type='submit' name='Submit' value='Submit' />
</form>

and here is the javascript:

$(document).ready(function () {

    $('#QuckCalc').validate({
        rules: {
            FirstSelect: {
                required: true;
            }
        }
        messages: {
            FirstSelect: "This is an error"
        }
    });

    alert($('#FirstSelect').rules());

})

This brings up what I expect but when I hit the Submit button, I get the
following error on jsfiddle:

{"error": "Shell form does not validate{'html_initial_name':
u'initial-js_lib', 'form': <mooshell.forms.ShellForm object at
0x2f37cd0>, 'html_name': 'js_lib', 'html_initial_id':
u'initial-id_js_lib', 'label': u'Js lib', 'field':
<django.forms.models.ModelChoiceField object at 0x8531610>,
'help_text': '', 'name': 'js_lib'}{'html_initial_name':
u'initial-js_wrap', 'form': <mooshell.forms.ShellForm object at
0x2f37cd0>, 'html_name': 'js_wrap', 'html_initial_id':
u'initial-id_js_wrap', 'label': u'Js wrap', 'field':
<django.forms.fields.TypedChoiceField object at 0x85316d0>,
'help_text': '', 'name': 'js_wrap'}"}

**Comment by Arkni** [comment]
You have some typos in your code:
* it's `QuickClac` not `QuckCalc`:
```js
      ↓ You missed an 'i' here
$('#QuckCalc').validate({
```
* Extra **`;`**
```js
FirstSelect: {
                  ↓
    required: true;
}
```

> JsFiddle does not support this combination of JQuery 2.1.4 and
> jqueryvalidation.js of 1.14.
> 
>     This combination breaks ALL of our selects on our website so it should
> be fairly easy to reproduce.
> 
> However, when I try to set up a JSFiddle, it breaks jsfiddle :

You need to provide the full url not just the file name.
For that, you have to use a CDN:

CDN | URL
-------- | -----------
cdnjs | https://cdnjs.cloudflare.com/ajax/libs/jquery-validate/1.14.0/jquery.validate.min.js
jsDelivr CDN | https://cdn.jsdelivr.net/jquery.validation/1.14.0/jquery.validate.min.js
Microsoft Ajax's CDN | http://ajax.aspnetcdn.com/ajax/jquery.validate/1.14.0/jquery.validate.min.js

Here is the updated fiddle and it's working as expected without any error: https://jsfiddle.net/Arkni/p80q2rLs/1/
